configs/sweeps/ilql_sweep.yml

```diff
@@ -0,0 +1,19 @@
+tune_config:
+  mode: "max"
+  metric: "metrics/sentiments"
+  search_alg: "random"
```

Comment:
Somewhere we should list all the options for these parameters(random vs bayesian)

Reply:
yeah, but we have only two options and I made two configs for each